jenkins/Pipeline.gy

```diff
@@ -66,9 +68,10 @@ pipeline {
         always {
             // Remove the pod which we just created and ran; remove any
             // dangling containers; and then remove any dangling images.
-            sh 'podman pod rm -f ${PB_POD_NAME} || true'
+            sh 'podman pod rm --force --ignore ${PB_POD_NAME} || true'
             sh 'podman container prune -f || true'
             sh 'podman image prune -f || true'
+            sh 'podman container cleanup --all --rm'
```

Comment:
I don't know if it matters, but it seems like we should do the container `cleanup` before doing the `prune`'s:  if the lack of clean-up prevents a container and/or image from being pruned, then we would prefer to do the clean up now rather than relegating the pruning to the next job.

Reply:
I don't know if it matters, too: I'd rather not deal with it here.